scripts/circle-ci/pr_changelog_status_check.sh

```diff
@@ -12,11 +12,17 @@ fi
 cd $(dirname $0)
 
 CURRENT_REF=$(git log -1 --format="%H")
-CHANGELOG_MODIFIED=$(git diff origin/develop...HEAD --name-only | grep $CHANGELOG -q >/dev/null)$?
+CHANGELOG_COMMIT_FLAG=$(git log origin/develop..HEAD --pretty=format:%B | grep '\[no release notes\]' -q >/dev/null)$?
```

Comment:
`-q` should be before the string to match standard conventions. Also applies to `--pretty-format` (it should be just after `log`)